src/numJewelsInStones.js

```diff
@@ -28,7 +28,21 @@
  * @return {number}
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  let counter = 0;
+
+  if (jewels === '') {
+    return 0;
+  }
+
+  const allJewels = jewels.match(/[a-z; A-Z]/gi);
```

Comment:
read about `i` in the `match`.
```suggestion
  const allJewels = jewels.match(/[a-z; A-Z]/gi);
```

Reply:
tnx, if 'i' dont need [A-Z], or if without 'i', need [A-Z]
good job)